### ISSUE TITLE
no method matching _erf(::Complex{BigFloat})

### ISSUE THREAD

**Issue by jherekhealy** [JuliaMath/SpecialFunctions.jl#416]
Hi,

This package is great, BigFloat and Complex are supported for erf and erfc, and the implementation is fast.
However, the combination of BigFloat and Complex is not supported. It would be great to add support for it, example:
erf(big(1.1im))

ERROR: MethodError: no method matching _erf(::Complex{BigFloat})
Closest candidates are:
  _erf(::Float64) at ~/.julia/packages/SpecialFunctions/9pXme/src/erf.jl:15

**Comment by stevengj** [comment]
Yes, this is not currently implemented. It’s not trivial to extend special-function algorithms to support arbitrary precision, unfortunately.
